frontend/src/types/api-platform/interfaces/fishbowl.ts

```diff
@@ -8,7 +8,7 @@
  */
 
 export interface Fishbowl {
-  id?: string;
+  "@id"?: string;
```

Comment:
This was manually changed, xD. We probably should have not only those types but ones generated from the actual graphql Schema.

Reply:
That also explains that the automatically generated types do not really work for the graphql queries.

Reply:
:+1: